packages/firestore/test/util/equality_matcher.ts

```diff
@@ -106,7 +105,7 @@ export function addEqualityMatcher(): void {
               /*showDiff=*/ true
             );
           } else if (originalFunction) {
-            originalFunction.apply(this, args);
+            originalFunction.call(this, expected, msg);
```

Comment:
Without these changes we were getting:

> Argument of type 'unknown[]' is not assignable to parameter of type '[unknown, unknown]'.
  Type 'unknown[]' is missing the following properties from type '[unknown, unknown]': 0, 1